Core/clim-core/presentations/typed-output.lisp

```diff
@@ -48,3 +48,156 @@
                          ,allow-sensitive-inferiors))
                    (,continuation)))
                (,continuation)))))))
+
+;;; XXX The spec calls out that the presentation generic function has
+;;; keyword arguments acceptably and for-context-type, but the
+;;; examples I've seen don't mention them at all in the methods
+;;; defined for present.  So, leave them out of the generic function
+;;; lambda list...
+(define-presentation-generic-function %present present
+  (type-key parameters options object type stream view
+   &key &allow-other-keys))
+
+(define-default-presentation-method present
+    (object type stream (view textual-view) &key acceptably for-context-type)
+  (declare (ignore for-context-type type))
+  (if acceptably
+      (let ((*print-readably* t))
+        (prin1 object stream))
+      (princ object stream)))
```

Comment:
There is another definition of this method in `standard-presentations.lisp` (or may `presentation-defs.lisp` in this commit).

Reply:
define-default-presentation-method and define-presentation-method are two very different macros, even expansion differ. I wouldn't touch it for now since this PR is about rearranging things.